tools/run_tests/xds_k8s_test_driver/framework/infrastructure/gcp/compute.py

```diff
@@ -40,8 +40,12 @@ class GcpResource:
     class ZonalGcpResource(GcpResource):
         zone: str
 
-    def __init__(self, api_manager: gcp.api.GcpApiManager, project: str):
-        super().__init__(api_manager.compute('v1'), project)
+    def __init__(self,
+                 api_manager: gcp.api.GcpApiManager,
+                 project: str,
+                 version: str = 'v1'):
+        # Version maybe also be v1alpha or v1beta
```

Comment:
For now we don't support `v1beta` in GcpApiManager. I think we can skip this comment.
```suggestion
```